en/docs/get-started/tutorials/create-your-first-webhook.md

```diff
@@ -31,151 +31,42 @@ First, let's add a Webhook component as follows:
     | **Name**        | `IssueTracking`       |
     | **Description** | `Track GitHub issues` |
 
-4. Click **Create**.
+4. Click **Create**.This takes you to the **Components** page.
 
-    The **Components** page opens as shown in the below image.
+5. On the **Components** page, click **Create** on the the **Webhook** card.
```

Comment:
- "the" is repeated.
- Let's avoid the term "card". How about "On the **Components** page, click **Create** under **Webhook**.

Reply:
We have used the term card commonly in the documentation. So left that as it is. other changes are done.

Reply:
"the" is still repeated.